src/frontend/src/lib/components/admin/ProductForm.svelte

```diff
@@ -0,0 +1,153 @@
+        placeholder="Title"
+        class="input mt-1 input-md block w-full rounded-lg border border-gray-700 focus:outline-none focus:border-indigo-500 focus:ring-indigo-500 sm:text-sm {isValidField(
+          $productForm.hasError('title.required')
+        )}"
+        bind:value={$formTitle.value}
+        {disabled}
+      />
+      {#if $productForm.hasError("title.required")}
+        <span class="text-red-500 text-sm ml-1 mt-1"> Product title is required </span>
+      {/if}
+    </div>
+    <div class="form-control w-full my-4">
+      <label for="description" class="block text-sm font-medium text-gray-700"
+        >Description: &nbsp;</label
+      >
+      <textarea
+        class="textarea mt-1 input-md block w-full rounded-lg border border-gray-700 focus:outline-none focus:border-indigo-500 focus:ring-indigo-500 sm:text-sm"
+        placeholder="Description"
+        bind:value={$formDescription.value}
+        {disabled}
+      />
+    </div>
+    <div class="form-control w-full my-4">
+      <label for="Category" class="block text-sm font-medium text-gray-700">Category: &nbsp;</label>
+      <select
+        class="select  w-full mt-1 rounded-lg border border-gray-700 focus:outline-none focus:border-indigo-500 focus:ring-indigo-500 sm:text-sm {isValidField(
+          $productForm.hasError('category.required')
+        )}"
+        bind:value={$formCategory.value}
+        {disabled}
+      >
+        <option value="default" disabled selected>Select category</option>
+        {#each $categories as c}
+          <option value={c[0]}>{c[1].name}</option>
+        {/each}
+      </select>
+      {#if $productForm.hasError("category.required")}
+        <span class="text-red-500 text-sm ml-1 mt-1"> Category is required </span>
+      {/if}
+    </div>
+    <div class="form-control w-full my-4">
+      <label for="price" class="text-sm font-medium text-gray-700">
+        Price (Satoshi): &nbsp;
+        <input
+          id="price"
+          alt="price"
+          type="number"
+          placeholder="1"
+          class="input mt-1 input-md block w-full rounded-lg border border-gray-700 focus:outline-none focus:border-indigo-500 focus:ring-indigo-500 sm:text-sm {isValidField(
+            $productForm.hasError('price.required')
+          )}"
+          bind:value={$formPrice.value}
+          {disabled}
+        />
+      </label>
+      {#if $productForm.hasError("price.required")}
+        <span class="text-red-500 text-sm ml-1 mt-1"> Price is required </span>
+      {/if}
+    </div>
+    <div class="form-control w-full my-4">
+      <label for="inventory" class="block text-sm font-medium text-gray-700">
+        Inventory: &nbsp;
+        <input
+          id="inventory"
+          alt="inventory"
+          type="number"
+          placeholder="1"
+          class="input mt-1 input-md block w-full rounded-lg border border-gray-700 focus:outline-none focus:border-indigo-500 focus:ring-indigo-500 sm:text-sm {isValidField(
```

Comment:
tu a aj v ostatných, kde sú čísla, by to asi chcelo pridať min atribút, aj v html aj ako validátor hore k required